packages/venia-concept/src/actions/cart/__tests__/asyncActions.spec.js

```diff
@@ -897,6 +897,32 @@ describe('toggleCart', () => {
         expect(dispatch).not.toHaveBeenCalled();
     });
 
+    test('toggleCart thunk opens the drawer and cart is empty', async () => {
+        getState.mockReturnValueOnce({
+            app: { drawer: 'cart' },
+            cart: { details: { items_count: 0 } }
+        });
+        await toggleCart()(...thunkArgs);
+
+        expect(dispatch).toHaveBeenNthCalledWith(1, expect.any(Function));
```

Comment:
While `expect.any(Function)` is a perfectly valid test case, it isn't very friendly for the reader because there's no context or intention.

Either of the following would help out the next person to read this:

**Add a comment**
```js
// toggleDrawer
expect(dispatch).toHaveBeenNthCalledWith(1, expect.any(Function));
```

or **Give it a semantic name**

```js
const toggleDrawer = expect.any(Function);
expect(dispatch).toHaveBeenNthCalledWith(1, toggleDrawer);
```

We're working on making this more consistent throughout the tests 👍

Reply:
I've applied the suggested changes.